Library/Homebrew/extend/pathname.rb

```diff
@@ -529,6 +529,12 @@ def unlink
     ObserverPathnameExtension.n += 1
   end
 
+  def mkpath
+    return if exist?
+    super
+    puts "mkdir #{self}" if ARGV.verbose?
+  end
```

Comment:
Oh, I see. I think the `return if exist?` can probably be skipped and do `puts "mkdir #{self}" if ARGV.verbose? && exist?` instead?